network-api/networkapi/news/tests_factory.py

```diff
@@ -0,0 +1,87 @@
+from django.test import TestCase
+from datetime import datetime
+
+from networkapi.news.factory import NewsFactory
+from networkapi.utility.utc import UTC
+
+utc = UTC()
+
+
+class TestNewsFactory(TestCase):
+    """
+    Test NewsFactory constructor
+    """
+
+    def test_news_creation(self):
+        """
+        Creating a news with the NewsFactory should not raise an exception
+        """
+
+        news = NewsFactory()
+
+        self.assertIsNotNone(news)
+
+    def test_news_creation_default(self):
+        """
+        Verify the default values applied to a News generated with the NewsFactory
+        """
+
+        news = NewsFactory()
+
+        self.assertIsNotNone(news.headline)
+        self.assertIsNotNone(news.outlet)
+        self.assertIsNotNone(news.date)
+        self.assertIsNotNone(news.link)
+        self.assertIsNotNone(news.excerpt)
+        self.assertIsNotNone(news.author)
+        self.assertIsNotNone(news.glyph)
+        self.assertEqual(news.featured, False)
+        self.assertLess(news.publish_after, datetime.now(tz=utc))
+        self.assertIsNone(news.expires)
+        self.assertEqual(news.is_video, False)
+        self.assertIsNotNone(news.thumbnail)
+
+    def test_news_is_featured_param(self):
+        """
+        The on_homepage kwargs should set featured to True
```

Comment:
`is_featured`?

Reply:
yup :D I changed it for consistency but forgot the comment because I was doing it manually >.<